src/goto-instrument/contracts/havoc_assigns_clause_targets.cpp

```diff
@@ -48,8 +48,8 @@ void havoc_assigns_clause_targetst::get_instructions(goto_programt &dest)
   for(const auto &pair : from_stack_alloc)
     havoc_if_valid(pair.second, havoc_program);
 
-  for(const auto &pair : from_heap_alloc)
-    havoc_if_valid(pair.second, havoc_program);
+  for(const auto &car : from_heap_alloc)
+    havoc_if_valid(car, havoc_program);
```

Comment:
This is expected since when replacing a call by a contract all objects to be havoced are collected in the `from_spec` set, and there is no malloc to collect objects from so from_heap_alloc will be empty